apps/analytics/scripts/seed.ts

```diff
@@ -0,0 +1,63 @@
+import dotenv from "dotenv";
+dotenv.config();
+
+import { faker } from "@faker-js/faker";
+import { db } from "~/db/db";
+import { eventsTable } from "~/db/schema";
+import { type EventInsertType } from "~/db/db";
+
+let installationIds: string[] = [];
+for (let i = 0; i < 20; i++) {
+  installationIds.push(faker.string.uuid());
+}
+
+const eventTypes = [
+  "AppSetup",
+  "ProtocolInstalled",
+  "InterviewStarted",
+  "InterviewCompleted",
+  "InterviewCompleted",
+  "DataExported",
+  "Error",
+];
```

Comment:
Once again, this should be reused from a SINGLE source of truth!!

Reply:
I didn't come up with this code sir it was copied from the old seed.js file, and here we are storing the values types in an array to be able to pass it to here `const type = faker.helpers.arrayElement(eventTypes);`, I know that `@codaco/analytics` package exports `EventTypes` as a type not a value, so maybe we should add another export that stores them in an array maybe?

Reply:
"In TypeScript, you can't directly convert a type to a value because types exist at compile time and are erased at runtime, while values exist at runtime"

Reply:
I didn't ask you to put this into a type - you derive the type from the variable in this case. What I was getting at is that these should only be defined once, and we have a definition of them in the analytics package already.

The point about this coming from existing code is irrelevant - you are doing the refactor here.